### ISSUE TITLE
Version for 1.12.2?

### ISSUE THREAD

**Issue by Proemmis** [Imprex-Development/orebfuscator#392]
### Description

Hello ! what is the version for mc 1.12.2 Java 8?

That's protocol lib and orefuscator 1.12.2 java 8 version? 

Thanks in advance :)

https://pastebin.com/zMznhm42

### Reproduction Steps

Hello ! what is the version for mc 1.12.2 Java 8?

That's protocol lib and orefuscator 1.12.2 java 8 version? 

Thanks in advance :)

https://pastebin.com/zMznhm42

### Expected Behaviour

Hello ! what is the version for mc 1.12.2 Java 8?

That's protocol lib and orefuscator 1.12.2 java 8 version? 

Thanks in advance :)

https://pastebin.com/zMznhm42

### Orebfuscator Dump

https://pastebin.com/zMznhm42

**Comment by Ingrim4** [comment]
You have to use Orebfuscator 5.4.4 for 1.12.2: https://github.com/Imprex-Development/orebfuscator/releases/tag/5.4.4
For ProtocolLib any version above 5.0.0 should be fine.

**Comment by Proemmis** [comment]
i thinks theses versions are not compatibles with my 1.12.2 version : https://pastebin.com/S1vcACkH

anyone have an idea? thanks :)

**Comment by Ingrim4** [comment]
Yeah your right I totally missed that you asked for Java 8 compatibility for Orebfuscator you could try 5.3.4 as that is the last Java 8 release we did.

For ProtocolLib either try [5.0.0](https://github.com/dmulloy2/ProtocolLib/releases/tag/5.0.0) or the [latest dev build](https://ci.dmulloy2.net/job/ProtocolLib/).

**Comment by Proemmis** [comment]
I think that dont works for me :( 

- protocolib 5.2.0 (latest) : https://pastebin.com/uWRNRU2L
- Protocolib 5.0.0 : https://pastebin.com/3EVtxYVB

how i can do ? :)

**Comment by Ingrim4** [comment]
Guess Orebfuscator 5.3.4 still had a bug for <= 1.14.x but version [5.3.2](https://github.com/Imprex-Development/orebfuscator/releases/tag/5.3.2) should work because it's using the old block property system.

Regarding ProtocolLib maybe try the latest dev build: https://ci.dmulloy2.net/job/ProtocolLib/lastSuccessfulBuild/artifact/build/libs/ProtocolLib.jar (version 731, you were using version 679)

**Comment by Proemmis** [comment]
that's dont work : https://pastebin.com/mFVr2pAC

**Comment by Ingrim4** [comment]
What type of Server are you using because for me ProtocolLib 5.1.0 and Orebfuscator 5.3.4 work fine in a clean spigot 1.12.2 server?

**Comment by Proemmis** [comment]
I use CatServer, (spigot bukkit and forge)

**Comment by Ingrim4** [comment]
I don't think this will work, because CatServer is known to have compatibility issues with ProtocolLib, even in newer versions. For reference, see: Luohuayu/CatServer#187. Another problem is that Orebfuscator was designed around Mojang's default block ID system, and CatServer doesn't seem to properly integrate its mod blocks into that system, which leads to errors.
